fluentui_core/src/main/java/com/microsoft/fluentui/theme/token/controlTokens/TabItemTokens.kt

```diff
@@ -63,14 +63,15 @@ open class TabItemTokens : IControlToken, Parcelable {
         )
     }
 
+
     @Composable
     open fun iconColor(tabItemInfo: TabItemInfo): StateColor {
         return when (tabItemInfo.fluentStyle) {
             FluentStyle.Neutral -> StateColor(
                 rest = FluentTheme.aliasTokens.neutralForegroundColor[FluentAliasTokens.NeutralForegroundColorTokens.Foreground3].value(
                     themeMode = FluentTheme.themeMode
                 ),
-                pressed = FluentTheme.aliasTokens.neutralForegroundColor[FluentAliasTokens.NeutralForegroundColorTokens.Foreground3].value(
+                pressed = FluentTheme.aliasTokens.brandForegroundColor[FluentAliasTokens.BrandForegroundColorTokens.BrandForeground1].value(
```

Comment:
Should we also consider changing text color as well in pressed state?

Reply:
changed the text color too in the pressed state.